imports/plugins/included/product-admin/client/components/ProductTable.js

```diff
@@ -163,101 +131,63 @@ function ProductTable({ onCreateProduct }) {
           />
         </Grid>
-      ) : null }
-      <Grid item sm={12} style={{ display: displayCard }}>
-        <Card raised>
-          <CardHeader
-            className={classes.cardHeaderTitle}
-            action={
-              <IconButton aria-label="close" onClick={() => setClosed(true)}>
-                <CloseIcon/>
-              </IconButton>
-            }
-            title={i18next.t("admin.importCard.title")}
-          />
-          <CardContent>
-            { files.length > 0 ? (
-              <Grid container spacing={1} className={classes.cardContainer}>
-                <Grid item sm={12}>
-                  {files.map((file, idx) => (
-                    <Chip
-                      variant="default"
-                      color="primary"
-                      label={file.name}
-                      key={idx}
-                      className={classes.leftChip}
-                      onDelete={() => handleDelete(file.name)}
-                    />
-                  ))}
-                </Grid>
-                <Grid item sm={12}>
-                  <Button
-                    variant="contained"
-                    color="primary"
-                    style={{ float: "right" }}
-                    onClick={() => importFiles(files)}
-                  >
-                    {i18next.t("admin.importCard.button")}
-                  </Button>
-                </Grid>
-              </Grid>
-            ) : (
-              <Grid container spacing={1} className={classes.cardContainer}>
-                <Grid item sm={12}>
-                  <Button
-                    {...getRootProps({ className: "dropzone" })}
-                    variant="contained"
-                    color="primary"
-                  >
-                    <input {...getInputProps()} />
-                    <ImportIcon className={classes.leftIcon}/>
-                    {i18next.t("admin.importCard.import")}
-                  </Button>
-                  <Typography variant="h5" display="inline" className={classes.helpText}>
-                    {i18next.t("admin.importCard.importHelpText")}
-                  </Typography>
-                </Grid>
-              </Grid>
-            )}
-          </CardContent>
-        </Card>
-      </Grid>
-      <Grid item sm={12} style={{ display: displayButton }}>
-        <Button
-          color="primary"
-          onClick={onCreateProduct}
-          variant="contained"
-        >
-          {i18next.t("admin.createProduct") || "Create product"}
-        </Button>
-      </Grid>
-      { isFiltered ? (
+      ) : null}
+      {(!isFiltered && !isTagSelectorVisible && !isFilterByFileVisible) && (
+        <Grid item sm={12} >
+          <Button
+            color="primary"
+            onClick={onCreateProduct}
+            variant="contained"
+          >
+            {i18next.t("admin.createProduct") || "Create product"}
+          </Button>
+        </Grid>
+      )}
+      {isFiltered ? (
```

Comment:
Refactor to `{isFiltered && (` and that way you don't have to use a ternary operation and return null for the else.

Reply:
refactored.